### ISSUE TITLE
Opening overlapping submenu on mobile triggers sub-item action

### ISSUE THREAD

**Issue by klues** [swisnl/jQuery-contextMenu#693]
Steps to reproduce:
1. go to https://swisnl.github.io/jQuery-contextMenu/demo/sub-menus.html in Chrome
2. open Developer tools (`Ctrl + Shift + I`)
3. use mobile device view (`Ctrl + Shift + M`)
4. Use a quite small device, e.g. Galaxy S5
5. Open the context menu of the demo by long tap
6. Select "Sub-Group" by tap
7. Alert opens -> sub-item of "Sub-Group" has been selected because it's positioned above the "Sub-Group" item and somehow the tap directly triggers the action of the overlapping element without even showing the submenu

**Comment by bbrala** [comment]
Hmm, yeah that is a problem. I checked your PR and it seems you add a delay after opening. Not sure if that is the way to get that fixed.

**Comment by klues** [comment]
Yeah, I've added a timeout of 100ms where the unintended action is ignored. For sure it's not a pretty solution, but it works. Feel free to change it to a more beautiful solution.

**Comment by klues** [comment]
Just tested again, and seems to be fixed in the meanwhile, couldn't reproduce it anymore.
